### DISCUSSION TITLE
How do I modify and redeploy the enterprise edition of bionic

### DISCUSSION THREAD

**Discussion by KivalM** [bionic-gpt/bionic-gpt#393] [Q&A]
Hi everyone.

I have made some development changes to Bionic (UI/Auth/features). How would I go about deploying this in place of the enterprise edition images on Git Hub? 

By this I mean, are there existing Dockerfiles that are used to build the ghcr.io images (referenced in "https://github.com/bionic-gpt/bionic-gpt/blob/main/crates/k8s-operator/config/bionic.yaml"), as I cannot find those files in the repo?

Thanks in advance.

**Comment by ianpurton** [comment]
Hi we use https://earthly.dev/ for builds and our `Earthfile` is in the root folder.

Are there changes you have made that we could commit to the main project?

**Reply by KivalM** [reply]
Thank you so much.
I completely missed that when reading through the project.

There are a few changes that may be relevant to contribute:
- invitations panel in the UI
- copy to clipboard for the chat UI

the rest is more personalization eg.
- a more ChatGPT esque UI

Let me know if any of these features interest you, so we can also have it maintained upstream.

**Reply by ianpurton** [reply]
Interested in all of those. 

Do you want to create pull requests for each (or all in 1) and I'll take a look?

**Reply by KivalM** [reply]
Great! I'll create them as separate PRs when I get a gap.
